app/models/concerns/featured_content_behavior.rb

```diff
@@ -0,0 +1,41 @@
+module FeaturedContentBehavior
+  extend ActiveSupport::Concern
+
+  def add_featured_content(solr_params)
+    featured_content_params = send("#{blacklight_params[:featured]}_content_params")
+    featured_content_params.each do |param|
+      solr_params[:fq] ||= []
+      solr_params[:fq] << param
+    end
+
+    solr_params
+  rescue NoMethodError
+    solr_params
+  end
+
+  private
+
+  def scanned_maps_content_params
```

Comment:
what do the Blacklight breadcrumbs look like for these queries?

Reply:
![screen shot 2018-01-24 at 11 47 18 am](https://user-images.githubusercontent.com/1656824/35350853-68a84b58-00fc-11e8-9349-a7a74ccec93b.png)